core/tst/software/aws/toolkits/core/parser/LambdaManifestValidatorTest.kt

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.core.parser
+
+import junit.framework.TestCase.assertFalse
+import junit.framework.TestCase.assertTrue
+import org.junit.Test
+import software.aws.toolkits.core.lambda.LambdaManifestValidator
+import java.io.FileInputStream
+
+class LambdaManifestValidatorTest {
+
+    @Test
+    fun isXmlParsing() {
+        val fileText = FileInputStream("./tst-resources/xmlSampleSuccess.xml")
```

Comment:
since these are in the tst-resources folder, you'd normally prefer to read these using the classloader with `getResourceAsStream`.

see existing usage in the schemas tests